### ISSUE TITLE
Update documentation about licences (CC-0 is also a valid licence choice)

### ISSUE THREAD

**Issue by zed-g** [TEI-Correspondence-SIG/CMIF#32]
Since we (at correspSearch) just got an inquiry about licence options, I would propose to update the relevant parts of <https://github.com/TEI-Correspondence-SIG/CMIF/blob/main/doc/documentation-en.md#information-about-the-digital-index-of-letters> to also include the licence option of `CC-0`. To my knowledge this is also an accepted licence choice for CMIF data and we already process and store CMIF data with this licence for correspSearch. It is also available as a licence option for our [CMIF Creator](https://correspsearch.net/de/cmif-creator.html).

I can provide the necessary changes to [`doc/documentation-en.xml`](<https://github.com/TEI-Correspondence-SIG/CMIF/blob/63564fa4bb9ede77c50b726d4d4bbb22ca39d57b/doc/documentation-en.xml>) (but I have no clue about your conversion-to-markdown routine).

**Comment by peterstadler** [comment]
Makes sense to me to also allow everything looser (or equivalent) than CC-BY-4.0.

**Comment by rettinghaus** [comment]
In general the documentation reflects what `correspSearch` needs, but not the customization (e.g., `licence` can be empty and its `target` attribute can point anywhere). It would be good if the documentation differentiated between the plain schema and application needs.

**Comment by zed-g** [comment]
> In general the documentation reflects what `correspSearch` needs, but not the customization (e.g., `licence` can be empty and its `target` attribute can point anywhere). It would be good if the documentation differentiated between the plain schema and application needs.

That's a good point, it's important to distinguish these two cases and be explicit about it. Either way, right now the documentation differs from the actual schema (and only mentions correspSearch once as far as I can see). If we leave the schema as is (with free choice of licences), we should at least strongly suggest/recommend CC-BY 4.0/ CC0/anything "looser" in the documentation I guess?

**Comment by StefanDumont** [comment]
I've updated the ODD and the documentation, so CC0 is valid now (in v1.1.0). 

In the case of the licence, I find it useful to standardize it in the format itself. As an _interchange_ format, the CMIF should ensure the re-usability of this file should and therefore be part of the format specification. 

However, a better distinction between format and application specifics is certainly worth considering. I'll create a new ticket here that only deals with this.
